app/controllers/planning_applications_controller.rb

```diff
@@ -327,10 +333,20 @@ def invalidation_notice_mail
   end
 
   def receipt_notice_mail
-    PlanningApplicationMailer.receipt_notice_mail(
-      @planning_application,
-      request.host,
-    ).deliver_now
+    applicant_and_agent_email(@planning_application).each do |user|
+      PlanningApplicationMailer.receipt_notice_mail(
+        @planning_application,
+        request.host,
+        user,
+      ).deliver_now
+    end
+  end
+
+  def applicant_and_agent_email(planning_application)
```

Comment:
this should live on the mailer, not on the controller: the controller says "send the email", and then the mailer can handle its logic of doubling the email